_posts/2017-10-05-denounce-segwit2x.md

```diff
@@ -0,0 +1,75 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org to denounce Segwit2x"
+permalink: /en/posts/denounce-segwit2x.html
+date: 2017-10-05
+---
+On 2017-10-11 at noon (UTC), Bitcoin.org is planning to publish a banner on every page of the site warning users about the risks of using services that will default to the so-called Segwit2x (S2X) contentious hard fork. S2X companies will be called out by name. To ensure that we only warn users against companies that will actually put user deposits at risk, we urge all companies to publicly clarify their stance before the above date, either by a highly-visible public statement or by commenting on Bitcoin.org issue #[1835](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1835) (or by doing both).
+
+In particular, we need to know that:
+
+1. The company will not under any circumstances list "Segwit2x" as "BTC" and/or "Bitcoin". Note that [Bitcoin is not ruled by miners](https://en.bitcoin.it/wiki/Bitcoin_is_not_ruled_by_miners), and miner actions cannot be used as a justification to redefine Bitcoin.
+2. The company will not by default *convert* user BTC to S2X-coins. Distributing "air-dropped" S2X-coins to users is acceptable, however.
+3. The company will continue to provide normal service to Bitcoin (ie. non-SegWit2x) users.
+
+Although bitcoin.org [condemns](/en/posts/hard-fork-policy) contentious hard fork attempts such as S2X, we consider it *tolerable* for companies to support S2X in ways that do not contradict the above three points, such as by supporting both Bitcoin and S2X simultaneously as separate cryptocurrencies.
+
+By default, we will be using the following list of companies known to support S2X in our warning:
```

Comment:
Suggest removing entire list of companies. It is impossible for bitcoin.org to keep track of each and every companies policies or intentions which can change at any time. Warn users of risks and why they exist.

Reply:
This post is about warning companies and giving them a chance to leave the NYA before a list is published in an alert, so I think it's very useful to explicitly name the companies so they know they're on the list.

These companies previously publicly signed their name in support of the NYA, so I don't see why we should be timid about republishing that information.

Reply:
Bitcoin.org should either develop objective rules for (similar to what is done for wallets) for services and corps, or leave well enough alone.

Bitcoin.org should not be about shaming corporations.

Reply:
IMO, this list is not about shaming companies (how can you shame a corporate entity anyway?  corporations don't have feelings) but rather preparing to alert users about the danger of using businesses that may destroy their bitcoins.

"Previously signed the NYA and has not withdrawn" seems like an objective enough criteria to me.

Furthermore, it seems to me like you simply don't like the idea of Bitcoin.org publishing a statement about this subject at all.  You have a right to that opinion, but perhaps it would be better for you to state your opposition directly rather than trying to nitpick it to death.

Reply:
I don't like the idea about calling companies out by name without specific risks cited that pertain exactly to that company's policy (we don't know how any of these companies will really handle 2x, how users funds will be at risk or not) and not offering users clear examples of risks and or benefits. I also don't like the idea of using bitcoin.org as some sort of  threat mechanism which this post clearly is.

I had hoped to try to salvage this and perhaps the "banner" that will follow, but it is begging to look unlikely that this is possible.

Reply:
The whole point is to name specific companies so that they a) clarify their stance on 2X and b) inform users of specific companies that they might want to avoid because those companies are likely to switch users from using Bitcoin to S2X-altcoin.